### PR TITLE
Fix hydra

### PR DESCRIPTION
**Pull request by ulrikstrid**
Hydra broke, this fixes it

### CODE REVIEW

File: ci/filter.nix [hunk 1]
```diff
@@ -113,6 +113,7 @@ let
     "ocaml-migrate-parsetree"
     "wasm"
     "pythonlib"
+    "camlp5"
```
Comment (ulrikstrid) [inline ci/filter.nix:116]: camlp5 seems to not be available for OCaml >= 4.13?

Comment (anmonteiro) [inline ci/filter.nix:116]: this is supposed to work, and has been building, except in 5.00

File: ci/filter.nix [hunk 2]
```diff
@@ -113,6 +113,7 @@ let
...
 
     # Broken on OCaml 5.00
     "fontconfig"
```
Comment (anmonteiro) [inline ci/filter.nix:119]: can you migrate these to the ocaml 5 extra ignores?

File: ci/filter.nix (hunk 1 above)
Comment (anmonteiro) [reply ci/filter.nix:116]: sure does https://github.com/anmonteiro/nix-overlays/blob/master/ocaml/default.nix#L171

File: ci/filter.nix (hunk 2 above)
Comment (ulrikstrid) [reply ci/filter.nix]: Done

**Pull request merged**